Удаление лишних спрайтов и звуков из порт-папки Проксимы

Exapsters opened the pull request:
:cl: Exapster
imagedel: Лишние спрайты и звуки в порт-папке были зачищены.
wip: Ведутся работы по портированию спрайтов и звуков из других сборок.
/:cl:

Exapsters committed:
Удаление кастомных звуков из порт-папки Проксимы

Re.

Exapsters committed:
Ликвидация бесполезных звуков из порт-папки Проксимы

Было удалено всё самое мало необходимое и ненужное вовсе. Оставлено лишь то, что потенциально может быть использовано на проекте.

Exapsters committed:
Восстановление утраченных файлов

Exapsters committed:
Восстановление последней части утраченных файлов

Exapsters committed:
Удаление лишних спрайтов из порт-папки Проксимы

Drakon341 reviewed (approved):
короче я типо посмотрел все 300+ файлов и всё хорошо. какая-то часть их используется, наверное. но это надо глубоко смотреть.

Exapsters commented:
> короче я типо посмотрел все 300+ файлов и всё хорошо. какая-то часть их используется, наверное. но это надо глубоко смотреть.

Пока идёт проверка. Должно выявить ошибки. Наши, в частности, Пораза, Плазмы и так далее - не трогал. Я их прекрасно помню.

Exapsters commented:
https://tenor.com/view/cat-dance-cat-dance-gif-20491618

Exapsters committed:
Удаление забытых файлов

Лишь те, о которых я недавно вспомнил. Ничего серьёзного или важного там не было.

polerno reviewed (approved):
proxima/icons/mob/weapons/64x_guns_left.dmi в этом файле лежит один из спрайтов моего концепта шприцемётной винтовки, но да пофигу. когда-нибудь верну и сделаю
proxima/icons/obj/JAMMER.dmi тут лежал концепт глушилки наушников, но опять же когда буду делать. тогда и верну
proxima/icons/obj/grenade.dmi тут также лежали концепты для гранат
proxima/icons/obj/guns/ballistic.dmi тут также лежали классные пушки, но опять для концепта
proxima/icons/obj/jesus.dmi а вот удалять икону Иисуса как-то не православно
proxima/icons/obj/weapons/items_and_weapons.dmi тут были классные спрайты для мага, увы их уже не вернуть
а вот трогать папку machines со звуками было рисковано экзап, там были звуки которые используются для фильтра
остальное похер, можно отправлять на мёрж

Exapsters commented:
> proxima/icons/mob/weapons/64x_guns_left.dmi в этом файле лежит один из спрайтов моего концепта шприцемётной винтовки, но да пофигу. когда-нибудь верну и сделаю proxima/icons/obj/JAMMER.dmi тут лежал концепт глушилки наушников, но опять же когда буду делать. тогда и верну proxima/icons/obj/grenade.dmi тут также лежали концепты для гранат proxima/icons/obj/guns/ballistic.dmi тут также лежали классные пушки, но опять для концепта proxima/icons/obj/jesus.dmi а вот удалять икону Иисуса как-то не православно proxima/icons/obj/weapons/items_and_weapons.dmi тут были классные спрайты для мага, увы их уже не вернуть а вот трогать папку machines со звуками было рисковано экзап, там были звуки которые используются для фильтра остальное похер, можно отправлять на мёрж

- В гранатах зачистил лишь самые неуместные спрайты.
- Баллистика также была зачищена, но не удалена полностью.
- Про Иисуса промолчу.
- Звуки машин я вернул давным давно.

polerno commented:
попробуй перезапустить ПР, возможно поможет

polerno commented:
@Exapsters давай, проксима мать ждёт тебя

Evie7056 commented:
Раз тесты зелёные, ничего лишнего не удалено.
Ну а Фолер может ещё раз залить все что ему нужно 🥴

Evie7056 commented:
Раз тесты зелёные, ничего лишнего не удалено.
Ну а Фолер может ещё раз залить все что ему нужно 🥴

Evie7056 reviewed (approved):
Ты положил мне браузер 300 удаленными файлами.
